docker/DockerBuild.sh

```diff
@@ -31,12 +31,13 @@ echo "CMake Source Directory: $CMAKE_SOURCE_DIR"
 echo "MiNiFi Package: $MINIFI_SOURCE_CODE"
 
 # Copy the MiNiFi source tree to the Docker working directory before building
+rm -rf $CMAKE_SOURCE_DIR/docker/minificppsource
```

Comment:
Excluding a path will not delete it in the target, even though we have the --delete flag said (this is the proper behaviour I think). However, this masked the error, because if you have already had the cmake directory copied, it left it as it was, just didn't update it, and the build succeeded. I had to clean up my clone to reproduce the issue.